plover/formatting.py

```diff
@@ -937,6 +972,20 @@ def _apply_space_char(text, space_char):
         return text
 
 
+def _apply_stitch_char(text, space_char, stitch_char):
+    if not text:
+        return text
+
+    new_chars = []
+    for i, j in zip(text, text[1:]):
```

Comment:
Wondering why you used a loop instead of list comprehension:

```python
space_char.join(
    [stitch_char.join(list(word)) for word in text.split(space_char)]
)
```